configure

```diff
@@ -0,0 +1,459 @@
+#!/usr/bin/env dmd -run configure
```

Comment:
Sounds like Andrei is on board. Someone recently had the same issue [1].

[1] http://forum.dlang.org/post/o5sd4e$14n1$1@digitalmars.com